src/main/java/com/zendesk/maxwell/MaxwellReplicator.java

```diff
@@ -134,7 +134,11 @@ private MaxwellAbstractRowsEvent processRowsEvent(AbstractRowEvent e) throws Sch
 
 		table = tableCache.getTable(e.getTableId());
 
-		if (table == null) {
+		if ( table == null && this.filter.hasTableBlacklist() ) {
+			LOGGER.warn(String.format("couldn't find table in cache for table id: %d, assuming blacklisted table", e.getTableId()));
+			return null;
```

Comment:
hmmm, so this is a tick unelegant.  how about this approach instead:

- in TableMapCache, maintain a separate `Set` of table ids that are blacklisted.  Then here you can definitely ask `tableCache.isBlacklisted(e.getTableId)` and leave the `SchemaSyncError` in place